src/utils.py

```diff
@@ -28,7 +28,36 @@ def create_directory_if_not_exists(path):
     return str(p_base_path)
 
 
-# Read latest chosen component
+# Store pipeline results
+def store_results(  # noqa: PLR0913
```

Comment:
I guess ideally this would be a component in the pipeline?
Fine for now to keep it like this.

Reply:
Didn't think about it but yes could be!